cocos/ui/UIListView.cpp

```diff
@@ -93,7 +93,7 @@ void ListView::updateInnerContainerSize()
         case Direction::VERTICAL:
         {
             size_t length = _items.size();
-            float totalHeight = (length - 1) * _itemsMargin;
+            float totalHeight = ((float)length - 1) * _itemsMargin;
```

Comment:
Thanks @enhhh 
size_t is evil here. we should change it with `int` or `ssize_t`.
Thoughts?